maven-model-transform/src/test/java/org/apache/maven/model/transform/ParentXMLFilterTest.java

```diff
@@ -33,30 +33,55 @@
 public class ParentXMLFilterTest
     extends AbstractXMLFilterTests
 {
-    private Function<XmlPullParser, ParentXMLFilter> filterCreator;
+    private Function<XmlPullParser, XmlPullParser> filterCreator;
 
     @BeforeEach
     void reset() {
         filterCreator = null;
     }
 
     @Override
-    protected ParentXMLFilter getFilter( XmlPullParser parser )
+    protected XmlPullParser getFilter( XmlPullParser parser )
     {
-        Function<XmlPullParser, ParentXMLFilter> filterCreator =
+        Function<XmlPullParser, XmlPullParser> filterCreator =
             (this.filterCreator != null ? this.filterCreator : this::createFilter);
         return filterCreator.apply(parser);
     }
 
-    protected ParentXMLFilter createFilter( XmlPullParser parser ) {
+    protected XmlPullParser createFilter( XmlPullParser parser ) {
         return createFilter( parser,
                 x -> Optional.of(new RelativeProject("GROUPID", "ARTIFACTID", "1.0.0")),
                 Paths.get( "pom.xml").toAbsolutePath() );
     }
 
-    protected ParentXMLFilter createFilter( XmlPullParser parser, Function<Path, Optional<RelativeProject>> pathMapper, Path projectPath ) {
+    protected XmlPullParser createFilter( XmlPullParser parser, Function<Path, Optional<RelativeProject>> pathMapper, Path projectPath ) {
         ParentXMLFilter filter = new ParentXMLFilter( parser, pathMapper, projectPath );
-        return filter;
+        return new FastForwardFilter( filter );
+    }
+
+    @Test
+    public void testWithFastForward()
```

Comment:
Just to check: would it make sense to also add a test for the opposite case, i.e. where the ParentXMLFilter is _not_ supposed to bypass other filters?